elephant/asset/asset.py

```diff
@@ -402,9 +407,6 @@ def _stretched_metric_2d(x, y, stretch, ref_angle, working_memory=None,
         array). This option should be used when there is not enough memory to
         allocate the full stretched distance matrix needed before DBSCAN.
         Default: None
-    verbose : bool, optional
-        Display progress bars and log messages.
-        Default: False
```

Comment:
The parameter is still there, so it should be listed as a deprecated non-functional parameter, also indicating how functionality is now handled.

Reply:
Maybe:
```
    verbose : deprecated
        This parameter is no longer functional. To control the verbosity of log messages, please use the module's logger that is based on the standard logging module. Logging is turned on by default (to level INFO). To restrict logging messages, use a higher logging level to WARNING or ERROR, e.g.,
            
            import logging
            from elephant.asset.asset import logger as asset_logger
            asset_logger.set_level(logging.WARNING)
```

Reply:
This is a good suggestion. I believe there is a directive for Sphinx so that the deprecation is visible.

```
.. deprecated:: 0.14.0
        This parameter is no longer .....
```

Reply:
See: [bdeb5e6](https://github.com/NeuralEnsemble/elephant/pull/570/commits/bdeb5e6fce95ce95c47e01ba29aa6ba0b5a15b93)